Essentials/src/main/java/net/essentialsx/api/v2/services/EssentialsService.java

```diff
@@ -0,0 +1,12 @@
+package net.essentialsx.api.v2.services;
+
+/**
+ * Provides basic information about a service registered by EssentialsX.
+ */
+public interface EssentialsService {
+    /**
+     * Gets the name of this service.
+     * @return The name of the service.
+     */
+    String getName();
```

Comment:
What exactly is this supposed to convey or be used for? The name of the service interface should identify the service, and ideally the implementation shouldn't be exposed to API users, so what does this represent?

Reply:
I don't remember why I added this, will remove